src/icon-manifest/supportedExtensions.ts

```diff
@@ -604,6 +604,7 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
     },
     { icon: 'watchmanconfig', extensions: ['.watchmanconfig'], filename: true, format: FileFormat.svg },
+    { icon: 'wasm', extensions: ['wasm'], languages: [languages.wasm, languages.wat], format: FileFormat.svg },
```

Comment:
Wondering if the `extensions` declaration has been done on purpose....

Reply:
Yes, it's explained in the PR description.

Reply:
And as usual, I missed it.